### PR TITLE
🦩fix secret type indent 🦩

### PR DESCRIPTION
**Pull request by eformat**
tab wrong. this causes issues when syncing from ArgoCD.

these are all fixes to get sync working for pelorus /deploy Chart from argocd.

https://github.com/rht-labs/ubiquitous-journey/pull/179

### CODE REVIEW

File: charts/deploy/templates/grafana-oauth-session-secret.yaml [hunk 1]
```diff
@@ -5,4 +5,4 @@ data:
 kind: Secret
```
Comment (etsauer) [inline charts/deploy/templates/grafana-oauth-session-secret.yaml:5]: @eformat I think this template is no longer used anyway. You can just delete it.

File: charts/deploy/charts/grafana-operator/templates/Grafana.yaml [hunk 2]
```diff
@@ -86,10 +86,8 @@ spec:
               properties:
                 ports:
                   type: array
-                  description: Override default ports
```
Comment (etsauer) [inline charts/deploy/charts/grafana-operator/templates/Grafana.yaml:89]: did you remove these descriptions intentionally?

Comment (eformat) [reply charts/deploy/charts/grafana-operator/templates/Grafana.yaml:89]: argocd-sync-grafana

yeah .. this is annoying - argocd sync fails with these descriptions there (1.6.1 argocd) .. CRD's are fiddly i know as far as sync is concerned will see if there is workaround on argocd side (ignoredifferences etc) ...

File: charts/deploy/templates/grafana-oauth-session-secret.yaml (hunk 1 above)
Comment (eformat) [reply charts/deploy/templates/grafana-oauth-session-secret.yaml:5]: @etsauer - no, looks like its still used here - https://github.com/redhat-cop/pelorus/blob/master/charts/deploy/templates/grafana.yaml#L61

File: charts/deploy/charts/grafana-operator/templates/Grafana.yaml (hunk 2 above)
Comment (eformat) [reply charts/deploy/charts/grafana-operator/templates/Grafana.yaml]: reverted these .. looking for argocd side change instead

**Comment by etsauer** [comment]
@eformat the suggestion for this was to just delete `charts/deploy/templates/grafana-oauth-session-secret.yaml` altogether rather than fix the indentation. This file is no longer used.

**Comment by eformat** [comment]
> @eformat the suggestion for this was to just delete `charts/deploy/templates/grafana-oauth-session-secret.yaml` altogether rather than fix the indentation. This file is no longer used.

yes, it is used ? here: https://github.com/redhat-cop/pelorus/blob/master/charts/deploy/templates/grafana.yaml#L61

**Comment by etsauer** [comment]
@eformat my mistake. I was looking at the wrong file. Can you tick the chart version number please?

**Pull request merged**